### DISCUSSION TITLE
Feature request: Icon color

### DISCUSSION THREAD

**Discussion by TimBeekman** [Clooos/Bubble-Card#552] [Feature requests]
It would be amazing to be able to set the icon color like in Mushroom cards, based on en Jinja2 template, like

{% if is_state('light.kitchen','on') %}
blue
{% else %}
none
{% endif %}

![image](https://github.com/Clooos/Bubble-Card/assets/89469564/3df0a133-1f62-4579-8af2-0cc015dd70a2)

**Comment by pdwonline** [comment]
You can do that with a custom style like this:
```
              .bubble-icon-container { background-color: ${state === 'on' ?  'blue' : 'var(--card-background-color) !important'}; }
```
